sklearn/tests/test_pipeline.py

```diff
@@ -332,6 +333,18 @@ def test_pipeline_methods_pca_svm():
     pipe.score(X, y)
 
 
+@pytest.mark.filterwarnings('ignore: default contamination parameter')  # 0.22
+def test_pipeline_score_samples_pca_lof():
+    iris = load_iris()
+    X = iris.data
+    # Test with PCA + OneClassSVM
+    clf = LocalOutlierFactor(novelty=True)
+    pca = PCA(svd_solver='full', n_components='mle', whiten=True)
+    pipe = Pipeline([('pca', pca), ('lof', clf)])
+    pipe.fit(X)
+    pipe.score_samples(X)
```

Comment:
please confirm the output is what you'd expect, at least to avoid regressions.

Reply:
I have added a check of the output : 
`assert_equal(list(pipe.score_samples(X[:2])), 
                       [-0.9564705258081416, -1.0371487183896932])`.

However, I have some float precision issues. The test works well with Linux py35_conda_openblas but not with the other settings (expected answer [-0.9564705258081414, -1.0371487183896926]). How do you usually deal with this kind of numerical issue ?